plugins/destination/kafka/client/read.go

```diff
@@ -16,14 +14,7 @@ const (
 )
 
 func (c *Client) ReverseTransformValues(table *schema.Table, values []any) (schema.CQTypes, error) {
-	switch c.pluginSpec.Format {
-	case FormatTypeCSV:
-		return c.csvReverseTransformer.ReverseTransformValues(table, values)
-	case FormatTypeJSON:
-		return c.jsonReverseTransformer.ReverseTransformValues(table, values)
-	default:
-		panic("unknown format " + c.pluginSpec.Format)
-	}
+	return c.filetype.ReverseTransformValues(table, values)
```

Comment:
embed client, drop this func

Reply:
Can't embed because there is no `BatchWrite` like the other plugins

Reply:
So, if you embed you don't have to implement a lot of functions, just an extra `BatchWrite`, right?
So shy don't just embed and ditch all transformers then, by implementing a singe `BatchWrite`?

Reply:
additionally, plugin-sdk/plugins/destination have both unimplemented stubs for managed & unmanaged writers to embed for easier impl